docs/launch-json.md

```diff
@@ -0,0 +1,104 @@
+# Visual Studio Code - launch.json sample
+
+If you want to debug either the `cluster_manager.py` or the `train_pipeline.py`
+using Visual Studio Code, you can take advantage on this `launch.json` sample
+to run it directly from the IDE.
+
+All you need to do is take care of filling in all the environment variables
+according to your cloud infrastructure setup.
+
+To know more about debugging in Visual Studio Code, visit
+[this link](https://code.visualstudio.com/Docs/editor/debugging).
+
+## launch.json file structure
+
+```json
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "01 - Create a new Cluster",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "env": {
```

Comment:
I agree about the duplication. But this approach was more dev-friendly because we have a single file that performs 4 actions, and each action needs a slightly different combination of environment variables filled.

For example:

`01 - Create a new cluster` is for `cluster_manager.py` and requires

- `DATABRICKS_DOMAIN`
- `DATABRICKS_ACCESS_TOKEN`
- **`DATABRICKS_CLUSTER_NAME_SUFFIX`**

While `03 - Start an existing Cluster` is also for `cluster_manager.py` and requires

- `DATABRICKS_DOMAIN`
- `DATABRICKS_ACCESS_TOKEN`
- **`DATABRICKS_CLUSTER_ID`**

If you also compare, for example `02 - Terminate a Cluster` and `03 - Start an existing Cluster`, they require the same set of environment variables, but `02` also receives the `--terminate` argument.

Having all these 5 different configuration makes it easy for people just to change the debugging configuration as they go, while if we have only one, we need to change variable values to match the action we want to take, each time.

![image](https://user-images.githubusercontent.com/874409/60743123-0e0f4980-9f25-11e9-87ce-683eefd20d3d.png)

Does it make sense?

I'm open to ideas and to change it to have a single configuration, if you still prefer.

Reply:
How about we leave the `ENV` section out of the launch.json so people just rely on the shell script to set them once?